yannickl88/features-bundle/1.0/etc/packages/features.yaml

```diff
@@ -0,0 +1,21 @@
+features:
+    tags:
+#        my.feature:
+#            request: ["beta", "on"]
+
+services:
```

Comment:
The classes it refers to, do not exist, they are merely examples. I do agree with keeping them in here, I'm just not entirely sure if the examples should be in here.